packages/block-editor/src/utils/pre-parse-patterns.js

```diff
@@ -65,7 +66,7 @@ export function usePreParsePatterns() {
 
 		handle = requestIdleCallback( callback );
 		return () => cancelIdleCallback( handle );
-	}, [ patterns ] );
+	}, patterns );
```

Comment:
Sorry I just got confused because I had inserter opened 🤔 It looks like every pattern preview in the Inserter triggers the `usePreParsePatterns` and all of them read all the patterns from the store. They don't seem to have a separate context.

Reply:
Gotcha, thanks! Removed the patterns from the block preview.